### PR TITLE
fastg to graph

### PR DESCRIPTION
**Pull request by cameronmartino**
edit (see doc of the new function for use etc...)

### CODE REVIEW

**Review by gwarmstrong — COMMENTED** [review]
Your stuff looked good with one or two _minor_ changes. I reorganized a little bit of the optim stuff and wrote a script that you should be able run almost directly. 

We need to decide what to do with the graph after we get it. E.g. presumably line 25 of scripts/run_optim.py should write contigs and graph score to a file?
Wanted to ask before we pursue further.

File: TesasSemble/fastg_graph.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    # map g2 nodes to g1 nodes
+    g2_only = [' '.join([map_g1_g2_names[y] 
+                         if y in map_g1_g2_names.keys() else y 
+                         for y in x.split(' ')]) for x in g2_only]
+
+    ## add both edges and nodes ##
+    G = RedBlueDiGraph()
+
+    # red and blue
+    nodes_ = list(set([j for i,_ in map_g1_g2
```
Comment (earmingol) [inline TesasSemble/fastg_graph.py:78]: I do not understand this part. Are you making g1 as a red graph and g2 a blue one?
Red edges in this case should be those edges that are in 100% of samples (or in a % to be defined). The blue ones are those that does not meet this.

File: TesasSemble/fastg_graph.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    for i in g1_only:
+        edges_ = i.split(' ')
+        for j in edges_[1:]:
+            edges[str(edges_[0])+'_'+str(j)] = Edge(str(edges_[0])+'_'+str(j), 
+                                                nodes_G[edges_[0]], nodes_G[j])
+    G.add_edges_from(list(edges.values()),['red']*len(list(edges.values())))
+
+    # blue (g2) only
+    edges = {}
+    for i in g1_only:
```
Comment (earmingol) [inline TesasSemble/fastg_graph.py:102]: This should be g2_only

Comment (cameronmartino) [reply TesasSemble/fastg_graph.py:102]: yes!

**Pull request merged**